liblangutil/ParserBase.cpp

```diff
@@ -85,6 +85,45 @@ void ParserBase::expectToken(Token _value, bool _advance)
 		m_scanner->next();
 }
 
+void ParserBase::expectTokenRecoveryDelete(Token _value, bool _advance)
+{
+	Token tok = m_scanner->currentToken();
+	if (tok != _value)
+	{
+		auto tokenName = [this](Token _token)
+		{
+			if (_token == Token::Identifier)
+				return string("identifier");
+			else if (_token == Token::EOS)
+				return string("end of source");
+			else if (TokenTraits::isReservedKeyword(_token))
+				return string("reserved keyword '") + TokenTraits::friendlyName(_token) + "'";
```

Comment:
I don't think we'll ever have keywords that contain a single quote, but we should probably consider having a function that safely quotes names for us (same with the 2 occurrences below).

Reply:
This is in code that was there from before. You are probably best at making the changes here, I am not sure I have the C++ chops to do it as well.

If you want to suggest code changes, please by all means do, and I'll test it along with the other stuff.

Or perhaps this should be a github issue and a separate PR?

Reply:
Aaaaaah. You copy'n'pasted those fragments from the `expectToken` implementation. That part wasn't shown in the diff, hence I thought it's **new** code. No need to go wild then ;-)